src/duplicacy_utils.go

```diff
@@ -300,6 +300,25 @@ func MatchPath(filePath string, patterns [] string) (included bool) {
              if matchPattern(filePath, pattern[1:]) {
                 return false
             }
+        } else if strings.HasPrefix(pattern, "i:") {
+            matched, err := regexp.MatchString(pattern[2:], filePath)
+            if err != nil {
+                LOG_ERROR("SNAPSHOT_MATCH", "Error during regexp match: %s - %v", pattern, err)
+            }
+            if matched {
+                LOG_TRACE("SNAPSHOT_MATCH", "Regex include filePath=\"%s\", pattern=\"%s\", matched=%t", filePath, pattern[2:], matched)
+                return true
+            }
+        } else if strings.HasPrefix(pattern, "e:") {
+            allIncludes = false
+            matched, err := regexp.MatchString(pattern[2:], filePath)
```

Comment:
The code from here down is duplicate except for the return value, which would be easy enough to abstract out.

Reply:
This is consistent with the existing pattern matching code for the + and - filter prefixes, so I don't feel that is needs to be reworked.